app/controllers/api/questions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Api::QuestionsController < ApplicationController
+  def index
+    render json: Question.all
+  end
+
+  def show
+    question = Question.find_by(id: params[:id])
```

Comment:
Use `find` instead of `find_by`

Reply:
That should suffice
```ruby
def show
  question = Question.find(params[:id])
  render json: question
end
```

Reply:
A question is related a exercise, so it will be something like this: 
```
@exercise.questions.find(params[:id])
```

 I can't create a question without a exercise.